front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -1221,7 +1222,7 @@ function SlackIntegration({
       {existingSelection.length ? (
         <>
           <div className="mt-6 text-sm text-element-700">
-            Your assistant will answer by default when @dust is mentioned in the
+            Your assistant will answer by default when @Dust is mentioned in the
```

Comment:
NIT NIT NIT: I would say "This assistant will answer" or "@name_of_the_assistant will answer"

Reply:
agreed that's better 👍

Reply:
agreed
@dust not @Dust

Reply:
Updated + added some other small improvements

Reply:
> @dust not @Dust

@Duncid  on slack it is `@Dust`

<img width="365" alt="Screenshot 2023-10-10 at 17 11 47" src="https://github.com/dust-tt/dust/assets/14199823/b7600162-3c8f-4cb0-8abe-8b19d0bb52bc">